front/app/api/initiative_images/keys.ts

```diff
@@ -1,19 +1,27 @@
+import { QueryKeys } from 'utils/cl-react-query/types';
+
+const baseKey = { type: 'image', variant: 'initiative' };
```

Comment:
Should 'image' be plural here?

Reply:
No, the type of the resource is singular (also of the back-end) for 99% of the cases. See comment above for the weird `initiatives` case in the action descriptors.